Corzbank.Services/UserService.cs

```diff
@@ -57,6 +57,9 @@ public async Task<Token> Login(UserForLoginModel userForLogin)
 
                 if(user.EmailConfirmed)
```

Comment:
do opposite. If the condition does not match your aim, return user.

Reply:
resolved